src/module/actor/actor-sheet.js

```diff
@@ -456,11 +402,11 @@ export class OseActorSheet extends ActorSheet {
 
     if (event.target.dataset.field === "value") {
```

Comment:
We could probably DRY up blocks like this so we only call `item.update` in one spot, like so:
```js
() => {
  const {field} = event.target.dataset;

  if (field !== 'value' && field !== 'max')
    throw new Error('Malformed update call when updating item quantity');
  
  return item.update({
    [`system.quantity.${field}]: parseInt(event.target.value)
  });
}
```
Not worth holding the PR up for, just a thought